include/cetl/any.hpp

```diff
@@ -294,10 +316,84 @@ class any : private detail::base_move<Footprint, Copyable, Movable>
 template <typename ValueType, typename Any, typename... Args>
 CETL_NODISCARD Any make_any(Args&&... args)
 {
-    return Any(cetl::in_place_type<ValueType>, std::forward<Args>(args)...);
+    return Any(in_place_type<ValueType>, std::forward<Args>(args)...);
+}
+
+/// \brief Constructs an any object containing an object of type T, passing the provided arguments to T's constructor.
+///
+/// Equivalent to `cetl::any(cetl::in_place_type<ValueType>, list, std::forward<Args>(args)...)`.
+///
+template <typename ValueType, typename Any, typename Up, typename... Args>
+CETL_NODISCARD Any make_any(std::initializer_list<Up> list, Args&&... args)
+{
+    return Any(in_place_type<ValueType>, list, std::forward<Args>(args)...);
 }
 
-// TODO: Add `any_cast` 1, 2 & 3.
+/// \brief Performs type-safe access to the contained object.
+///
+/// \param operand Target any object.
+/// \return Returns `std::static_cast<ValueType>(*cetl::any_cast<const U>(&operand))`,
+///     where let `U` be `std::remove_cv_t<std::remove_reference_t<ValueType>>`.
+///
+template <typename ValueType, typename Any>
+ValueType any_cast(const Any& operand)
+{
+    using RawValueType = typename std::remove_cv<typename std::remove_reference<ValueType>::type>::type;
```

Comment:
It is best to use `_t`/`_v` helpers where available.

```suggestion
    using RawValueType = std::remove_cv_t<std::remove_reference_t<ValueType>>;
```